README.md

```diff
@@ -27,9 +27,9 @@ See [Google Sign-In JavaScript
 API](https://developers.google.com/identity/sign-in/web/reference?hl=ca)
 for more details
 
-#### Type Gapi.ux_values
+#### Type `Gapi.ux_values`
 
-The arguments untitled "ux_mode" are "string" that represent the "UX"
+The arguments untitled `ux_mode` are "string" that represent the "UX"
```

Comment:
What does it mean?

Reply:
> What does it mean?

I don't really understand either, the documentation only said "The UX mode to use for the sign-in flow. By default, it will open the consent flow in a popup. Valid values are popup and redirect."

Reply:
I was asking about the meaning of *The arguments untitled `ux_mode`*.

Reply:
And that was the only description that I can find on the API, but I done some external research since and I think I understand now: I will rewrite this section soon.

Reply:
It should be more clear now I hope.